src/applications/vaos/services/appointment/index.js

```diff
@@ -633,6 +633,23 @@ export function isVideoHome(appointment) {
   );
 }
 
+/**
+ * Method to get patient video instruction
+ * @param {Appointment} appointment A FHIR appointment resource
+ * @return {string} Returns patient video instruction title and exclude remaining data
+ */
+
+export function getPatientInstruction(appointment) {
+  if (appointment?.patientInstruction.includes('Medication Review')) {
+    return 'Medication Review';
+  }
+  if (appointment?.patientInstruction.includes('Video Visit Preparation')) {
+    return 'Video Visit Preparation';
+  } else {
```

Comment:
Is there a reason this `else` is chained to video visit preparation? If we want this method to return `null` if none of the conditions are met maybe we could remove the `else` entirely and just return `null` after the `if` statements.